realm-annotations/src/main/java/io/realm/annotations/RealmNamingPolicy.java

```diff
@@ -105,8 +105,8 @@
      * if set in {@link RealmClass#fieldNamingPolicy}, the module policy will still apply to field
      * names.
      * <p>
-     * If two modules disagree on the policy and one of them is {@code NO_POLICY}, the other one
-     * will be chosen without an error being thrown.
+     * If two modules disagree on the policy and one of them is {@code NO_POLICY}, the other will
```

Comment:
maybe `the other module's policy`

But what happens if there are 3 `@RealmModule`s and they all have conflicting policy? Which one will override `NO_POLICY`?

Reply:
For each class you find all modules it is part of, then you compare the policy between all these 3, so it will throw. The implementation is slightly different, but the effect is the same.